connector/pom.xml

```diff
@@ -357,6 +357,16 @@
                 </executions>
                 <configuration>
                     <addScalacArgs>-target:jvm-1.8</addScalacArgs>
+                    <args>
+                        <arg>-Ywarn-unused</arg> <!-- if you need exactly RemoveUnused -->
```

Comment:
what this means?

Reply:
If there are unused dependencies, this flag will report them. We eventually will add scalafix to fix code smells, we had to add this